src/layouts/LinkReport/LinksReport.tsx

```diff
@@ -257,7 +264,14 @@ const LinkContent = ({ brokenLinks }: { brokenLinks: RepoError[] }) => {
       alignContent="normal"
       alignItems="flex-start"
     >
-      <VStack spacing="0.5rem" m="1rem" textStyle="body-1" alignItems="start">
+      <VStack
```

Comment:
Lgtm, minor feedback - I thought the sidebar on left looks somewhat plain and different? Also is there a max viewport set for the horizontal width

Reply:
what did you mean by max width ya? the page is not horizontally scrollable, wondering what is the context behind this question